addon/components/r-avatar/component.js

```diff
@@ -18,16 +18,12 @@ export default Component.extend({
     'x-large:icon-x-large'
   ],
 
-  options: {
-    size: ['small', 'medium', 'large', 'x-large']
-  },
-
-  defaults: {
-    size: 'medium'
-  },
-
   init() {
     this._super(...arguments);
+    this.options = {
+      size: ['small', 'medium', 'large', 'x-large']
```

Comment:
Because of https://dockyard.com/blog/2015/09/18/ember-best-practices-avoid-leaking-state-into-factories